test/Microsoft.AspNet.Mvc.FunctionalTests/Compiler/Resources/MvcTagHelpersWebSite.MvcTagHelper_Home.Script.html

```diff
@@ -7,34 +7,34 @@
 </head>
 <body>
     <h2>Script tag helper test</h2>
-    <script src="/site.js" data-foo="foo-data1">
+    <script src="/site.js" data-foo="foo-data1" title="&lt;the title>">
         // Regular script with comment in body, and extra properties.
     </script>
 
-    <script src="/blank.js" data-foo="foo-data2">
+    <script data-foo="foo-data2" title="&lt;the title>" src="/blank.js?a=b&amp;c=d">
         // TagHelper script with comment in body, and extra properties.
     </script>
-<script>(false||document.write("<script src=\"/site.js\" data-foo=\"foo-data2\"><\/script>"));</script>
```

Comment:
`src=\"/site.js\"` here indicates a slightly different bug in previous version of the `<script/>` tag helper: unlike `<link/>` tag helper, did not JavaScript-encode fallback URLs.